app/2.0/docs/devguide/encapsulated-style.md

```diff
@@ -0,0 +1,639 @@
+
+```html
+<!-- Document-level stylesheet -->
+<style>
+	p { 
+		font-family: sans-serif;
+		color:blue;
+    }
+</style>
+<dom-module id='x-foo'>
+  <template>
+    <!-- Encapsulated, element-level stylesheet -->
+    <style>
+		p {
+			font-family: sans-serif;
+			color:green;
+		}
+		</style>
+		<p>I'm green.</p>
+  </template>
+</dom-module>
+
+<p>I'm blue.</p>
+<p><x-foo></x-foo></p> 
+```
+
+### Styling the host element
+
+
+The element to which shadow DOM is attached is known as the host. To style the host, use the `:host` selector.
+
+Inheritable properties of the host element will inherit down the shadow tree, where they apply to the shadow children.
+
+[See it on Plunker](http://plnkr.co/edit/7771DvsQ3iPWnn2gEIf8?p=preview)
+
+```html
+<dom-module id="x-foo">
+	<template>
+		<style>
+		:host {
+			font-family: sans-serif;
+			color: green;
+			display: block;
+			border: 1px solid;
+		}	
+	</style>
+	<p>I'm green.</p>
+	<div>I'm green too.</div>
+	<span>We're all green...</span>
+	</template>
+</dom-module>
+<x-foo></x-foo>    
+```
+
+You can also style the host element from outside - for example, using a type selector:
+
+[See it on Plunker](http://plnkr.co/edit/AHXFX0zeQTbO2rGELTbS?p=preview)
+
+```html
+x-foo {
+	background-color: blue;
+}
+```
+
+#### Using selectors 
+
+You can use CSS selectors to determine when and how to style the host. In this code sample:
+
+* The rule `:host { font-family: sans-serif; }` always matches `<x-foo>` elements
+* The rule `:host(.blue) {color: blue;}` matches `<x-foo>` elements of class `blue`
+* The rule `:host(.red) {color: red;}` matches `<x-foo>` elements of class `red`
+* The rule `:host(:hover)` matches `<x-foo>` elements when they are hovered over
+
+[See it on Plunker](http://plnkr.co/edit/FsXnCAz65SR6fZ7YKuy6?p=preview)
+
+```html
+<dom-module id="x-foo">
+	<template>
+		<style>
+			:host { font-family: sans-serif; }
+			:host(.blue) {color: blue;}
+			:host(.red) {color: red;}
+			:host(:hover) {color: green;}
+		</style>
+		<p>Hi, from x-foo!</p>
+	</template>
+</dom-module>
+<x-foo class="blue"></x-foo>
+<x-foo class="red"></x-foo>
+```
+
+Descendant selectors after `:host` match elements in the shadow tree. In this example, the CSS selector applies to any `p` element in the shadow tree if the host has class "warning":
+
+[See it on Plunker](http://plnkr.co/edit/MRN9blKg6A3w8G0RkyJD?p=preview)
+
+```html
+<dom-module id="x-foo">
+	<template>
+		<style>
+		:host(.warning) p {
+			color:red;
+		}
+		</style>
+    <p>Make this text red if x-foo has class "warning", and black otherwise.</p>
+	</template>
+</dom-module>
+<x-foo class="warning"></x-foo>
+<x-foo></x-foo>
+```
+
+Styling with the `:host` selector is one of two instances where rules inside a shadow tree can affect an element outside a shadow tree. The second instance uses the `::slotted()` syntax to apply styling rules to content populated from the main DOM at runtime.
```

Comment:
"from the main DOM" is not quite right, because the distributed children could be coming from a parent shadow tree. I think you should say "distributed children" here. (Maybe reference the shadow DOM concepts chapter for a discussion, although that definition is kind of buried in the "Shadow DOM and composition" section.